otelfiber/test/fiber_test.go

```diff
@@ -1,4 +1,4 @@
-package otelfiber
+package test
```

Comment:
why did you change name of the package

Reply:
To follow https://github.com/open-telemetry/opentelemetry-go/blob/main/bridge/opencensus/test/bridge_test.go
But, I guess to follow that entirely test should be in a separate module to avoid including trace test.

> If users do not want to include a dependency on the default SDK it is recommended to run integration tests in their own module to isolate the dependency (see go.opentelemetry.io/otel/bridge/opencensus/test as an example).

Reply:
> To follow https://github.com/open-telemetry/opentelemetry-go/blob/main/bridge/opencensus/test/bridge_test.go But I guess to follow that entirely, test should be in a separate module to avoid including tracetest.
> 
> > If users do not want to include a dependency on the default SDK it is recommended to run integration tests in their own module to isolate the dependency (see go.opentelemetry.io/otel/bridge/opencensus/test as an example).

Got it. Can you make it otelfiber_test

Reply:
sure, but I wonder if we should make it a separate module.